types/types.go

```diff
@@ -499,6 +550,15 @@ func HashBytes(b []byte) Hash256 { return blake2b.Sum256(b) }
 // implementation whose constructor returns a concrete type.
 var hasherPool = &sync.Pool{New: func() interface{} { return NewHasher() }}
 
+// HashObject computes the hash of the object using Sia's hash function.
+func HashObject(obj EncoderTo) Hash256 {
```

Comment:
is this being used anywhere? It forces an allocation and usually we're hashing multiple objects together, not just one

Reply:
Yes, ephemeral accounts hash and sign the withdrawal request